sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py

```diff
@@ -88,7 +88,6 @@ def __init__(
         table: Optional[str] = None,
         created_timestamp_column: Optional[str] = "",
         field_mapping: Optional[Dict[str, str]] = None,
-        date_partition_column: Optional[str] = None,
```

Comment:
intentional?

Reply:
Yes, this isn't used in trino source at all, I think our plan was to kill this anyways.